src/utility/algod.utility.ts

```diff
@@ -91,3 +91,8 @@ export const optInDaoApp = async (
 		throw error;
 	}
 };
+
+export const convertToHex = (address: string) => {
+	const pk = algosdk.decodeAddress(address);
+	return Buffer.from(pk.publicKey).toString("hex");
+};
```

Comment:
I didn't understand the use of this function, what does it do.?

Reply:
It converts the address into a hexadecimal address. The address is stored in PostgreSQL in hexadecimal encoding.